mitodl/micromasters
# Add REST API to create channels

#### What are the relevant tickets?
Part of #3475

#### What's this PR do?
Adds a REST API to create channels on the open-discussions server and store an associated percolate query used to subscribe users to that channel. This PR does not yet subscribe those users.

#### How should this be manually tested?
Go to `/api/v0/channels/` and create a channel. Use `{}` for the percolate query. You should see a response with an updated percolate query with the program selector. If you browse the reddit server you should see the new channel.
